ecs-cli/modules/utils/compose/ecs_params_reader.go

```diff
@@ -39,6 +41,28 @@ type ContainerDef struct {
 	Essential bool `yaml:"essential"`
 }
 
+// Non-TaskDefinition specific parameters
+type RunParams struct {
```

Comment:
Nit: Applies to multiple names here: https://github.com/golang/go/wiki/CodeReviewComments#initialisms
(This is also the source for other similar comments I have made)